packages/vscode-extension/src/plugins/network/network-plugin.ts

```diff
@@ -4,16 +4,19 @@ import { WebSocketServer, WebSocket } from "ws";
 import { Devtools } from "../../project/devtools";
 import { ToolKey, ToolPlugin } from "../../project/tools";
 import { extensionContext } from "../../utilities/extensionContext";
+
+import { Logger } from "../../Logger";
 import { NetworkDevtoolsWebviewProvider } from "./NetworkDevtoolsWebviewProvider";
 
 export const NETWORK_PLUGIN_ID = "network";
 
-let initialzed = false;
-function initialize() {
-  if (initialzed) {
+let initialized = false;
+async function initialize() {
```

Comment:
why is this changed to async? There are no awaits added in this PR for this method